package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "template-widget",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --config webpack.config.js --watch",
+    "start": "webpack-dev-server --hot --config webpack.config.js --port 9008",
+    "build:prod": "webpack --config webpack.config.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@ada-support/ada-widget-sdk": "^0.0.7"
+  },
+  "devDependencies": {
+    "css-loader": "^3.6.0",
+    "html-webpack-plugin": "^4.3.0",
+    "style-loader": "^1.2.1",
```

Comment:
I forgot to remove the `css-loader` and `style-loader` packages. We don't need them if we're not loading css in the build